web/packages/teleterm/src/ui/DocumentCluster/ActionButtons.tsx

```diff
@@ -307,3 +307,29 @@ function AppButton(props: {
     </ButtonBorder>
   );
 }
+
+export function AccessRequestButton(props: {
+  isResourceAdded: boolean;
+  requestStarted: boolean;
+  onClick(): void;
+}) {
+  return props.isResourceAdded ? (
+    <ButtonPrimary
+      textTransform="none"
+      width="124px"
+      size="small"
+      onClick={props.onClick}
+    >
+      Remove
+    </ButtonPrimary>
+  ) : (
+    <ButtonBorder
+      textTransform="none"
+      width="124px"
+      size="small"
+      onClick={props.onClick}
+    >
+      {props.requestStarted ? '+ Add to request' : '+ Request Access'}
```

Comment:
```suggestion
      {props.requestStarted ? '+ Add to request' : '+ Request access'}
```

This way the capitalization matches other states. But idk how it's spelled out in the Web UI.

Reply:
Actually, in the Web UI it is  '+ Add to request'  and '+ Request Access' :)
But in Connect we usually use lowercase (for example in the additional actions) so I change it to lowercase.